source/wifi/wifi_hal.c

```diff
@@ -3702,15 +3708,15 @@ INT wifi_getApWpaEncryptionMode(INT apIndex, CHAR *output_string)
     }
 
     if((strcmp(buf,"3")==0))
-        strcpy(params.name,"rsn_pairwise");
```

Comment:
Its is valid change. We should have handled this as part of this commit 54833c023ddff2d8daefe6ce09041c6069c14a42 where we changed char name[64]-> char *name. 
Requesting you to make similar change in wifi_setApBasicAuthenticationMode API also.